docs/git-guide.md

```diff
@@ -1255,6 +1255,13 @@ $ git rebase --continue
 Applying: test change for docs
 ```
 
+Note: You can see recent changes made to a file by running the following
+commands:
+```
+git fetch upstream
+git log -p upstream/master -- <file>
```

Comment:
that's a clearer way to write it. was struggling with how to make `pathToFile/file` work. :D